components/vm/25-vms.tf

```diff
@@ -38,6 +38,7 @@ module "virtual-machines" {
 
   custom_script_extension_name = "HMCTSVMBootstrap"
   tags                         = module.ctags.common_tags
+  xdr_tags                     = "${local.xdr_tags.activityName},${local.xdr_tags.application},${local.xdr_tags.org},${local.xdr_tags.server},${local.xdr_tags.env}"
```

Comment:
why not put the `join` from the last commit directly as the value here? Its much cleaner

Reply:
I was testing something but it appear twice